tests/gold_tests/pluginTest/xdebug/x_remap/out.gold

```diff
@@ -54,15 +56,17 @@ X-Remap: from=http://one/, to=http://127.0.0.1:SERVER_PORT/
 
 {'xDebugProbeAt' : '``
    'captured':[{'type':'response', 'side':'server', 'headers': {
+	'Start-Line' : 'HTTP/1.1 200 OK',
```

Comment:
This is called a Status Line, but again I don't think it really matters.  :)
https://tools.ietf.org/html/rfc7230#section-3.1.2

Reply:
The tutorial-type documents I found indicated that "start line" was the general term for both  "request line" (for requests) and "status line" (for responses).  I suppose it might be better to use the specific name for requests versus responses.  ATS7 does not use JSON for the header data.  I don't know who drove the change to JSON so I don't know who to ask what's best to do with this.

Reply:
https://www.tutorialspoint.com/http/http_messages.htm